org.zotero.Zotero.appdata.xml

```diff
@@ -24,7 +24,7 @@
   <launchable type="desktop-id">org.zotero.Zotero.desktop</launchable>
   <update_contact>guillaumepoiriermorency@gmail.com</update_contact>
   <releases>
-    <release version="5.0.66" date="2019-04-12"></release>
```

Comment:
you should ideally have multiple release tag lines, just leave the old one behind next time.